pkg/internal/resources/apiserver/apiserver.go

```diff
@@ -61,6 +62,10 @@ func Manifests(c Config) ([]unstructured.Unstructured, error) {
 		}
 	}
 	const AuthModeEnv = "AUTHENTICATION_MODE"
+	var logLevel int
+	if c.Spec.LogLevel != nil {
```

Comment:
I have checked the webhook implementation, webhook(defaulting of loglevel) will only be triggered in CREATE request, what if someone issues a UPDATE request with LogLevel nil? It will be set to "0". But I think it should still be the loglevel of Operator.

Reply:
> But I think it should still be the loglevel of Operator.

I disagree, it should be default log level defined in the Kubecarrier object

Reply:
yeah, you are right, it should be default to Kubecarrier loglevel.

Reply:
working on getting logLevel from `Kubecarrier` object

Reply:
done